test/integration/host_js_test.rb

```diff
@@ -340,6 +340,21 @@ class HostJSTest < IntegrationTestWithJavascript
       assert_current_path hosts_path
       assert_empty(page.driver.cookies['_ForemanSelectedhosts'])
     end
+
+    test 'redirect js' do
+      visit hosts_path
+      page.find('#check_all').trigger('click')
+
+      # Ensure and wait for all hosts to be checked, and that no unchecked hosts remain
+      assert page.has_no_selector?('input.host_select_boxes:not(:checked)')
+
+      # Hosts are added to cookie
+      host_ids_on_cookie = JSON.parse(CGI.unescape(page.driver.cookies['_ForemanSelectedhosts'].value))
+      assert(host_ids_on_cookie.include?(@host.id))
+
+      page.execute_script("build_redirect('#{select_multiple_environment_hosts_path}')")
```

Comment:
what do you think about instead of execute `build_redirect` directly,  we can stub `multiple_actions` to return something like `[[_('a redirect action'), some_path, false]]`, and then the `build_redirect` should be called ?